src/backends/onnx.js

```diff
@@ -27,7 +27,10 @@ export const executionProviders = [
     'wasm'
 ];
 
-if (typeof process !== 'undefined' && process?.release?.name === 'node') {
+if (
+    (typeof process !== 'undefined' && process?.release?.name === 'node') ||
+    (typeof navigator !== 'undefined' && navigator.product === 'ReactNative')
```

Comment:
Can we add `onnxruntime-react-native` to optional deps instead of using alias like [the example](https://github.com/hans00/react-native-transformers-example/blob/main/babel.config.js#L14)?

Reply:
Maybe use `import(package)`?